app/views/layouts/_navbar.html.erb

```diff
@@ -10,12 +10,12 @@
 
   <div class="collapse navbar-collapse" id="global-navbar-content">
     <ul class="navbar-nav mr-auto">
-      <li class="nav-item <%= 'active' unless defined?(activenav) && (activenav == 'download') %>">
-        <a class="nav-link" href="/explore"><%= _("Software") %></a>
-      </li>
       <li class="nav-item <%= 'active' if defined?(activenav) && (activenav == 'download') %>">
         <a class="nav-link" href="/distributions"><%= _("Download") %></a>
       </li>
+      <li class="nav-item <%= 'active' if defined?(activenav) && (activenav == 'software') %>">
```

Comment:
This is not only swapping the tabs. It looks like you are also fixing a bug when `activenav` is not defined. If so, I would add it to the commit.

Reply:
after taking a closer look, this is not solving a bug... Before software was selected by default, now only if it is assigned. This still works because `activenav` is assigned in `app/views/layouts/application.html.er` to software by default. But I would keep the check here as well. Maybe you find this more readable:

```ruby
if !defined?(activenav) || (activenav == 'software')
```